corehq/apps/hqadmin/tests/test_stale_data_in_es.py

```diff
@@ -193,6 +196,20 @@ def call():
         self._send_cases_to_es([case])
         self._assert_in_sync(call())
 
+    def _test_case_missing_date(self):
+        def call():
+            return self._stale_data_in_es('case')
+        form, (case,) = self._submit_form(self.project.name, new_cases=1)
+
+        pg_modified_on = case.server_modified_on
+        case.server_modified_on = None
+        self._send_cases_to_es([case], refetch_doc=False)
+        case.server_modified_on = pg_modified_on
+
+        self._assert_not_in_sync(call(), rows=[
```

Comment:
Why is the `call` function needed? Seems like it would be clearer to just call `_stale_data_in_es` where it's needed.
```suggestion
        form, (case,) = self._submit_form(self.project.name, new_cases=1)

        pg_modified_on = case.server_modified_on
        case.server_modified_on = None
        self._send_cases_to_es([case], refetch_doc=False)
        case.server_modified_on = pg_modified_on

        data = self._stale_data_in_es('case')
        self._assert_not_in_sync(data, rows=[
```

Reply:
That is a great question. It is how the other tests were set up, so I left it that way for consistency, but could be convinced to change this (and the others, since I think the consistency is important). Going to merge for now, but if you think it is worth changing all of them, happy to follow up with that.

Reply:
I think it would be easier to read the tests if they were changed.